File: Cuke4Duke/trunk/NoChangeParking/src/main/java/com/sqs/nochangeparking/core/GreetingServer.java
package com.sqs.nochangeparking.core;

public class GreetingServer {
	
	private static GreetingServer instance;
	
	private GreetingServer() {
		
	}

	public static GreetingServer getInstance()
	{
		if (instance == null)
			instance = new GreetingServer();
		return instance;
	}
	
	public static String getApplicationName() {
		return "NoChangeParking";
	}
	
	public static String sayHello(String name) {
		if (name.equals("")) {
			return "Hello Stranger";
		} else {
			return "Hello " + name;
		}
	}
	
}

File: DotNetFitNesse/trunk/NoChangeParking/src/NoChangeParking.Core/GreetingServer.cs
using System;

namespace NoChangeParking.Core
{
	public class GreetingServer
	{
		public string SayHello(String name) {
			if (name == "") {
				return "Hello Stranger";
			} else {
				return "Hello " + name;
			}
		}
	}
}
